Header.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from "react-native";
+
+export const Headers = () => (
+  <View style={styles.container}>
+    <Text style={styles.left}>Cancel</Text>
+    <Text style={styles.main}> New task </Text>
```

Comment:
```suggestion
    <Text style={styles.main}>New task</Text>
```

These spaces aren't neeeded.